### PR TITLE
Create a new topic to cover Transferring Tokens using a Deploy File

### PR DESCRIPTION
**Pull request by GloreenS**
### Changes

- Added a flowchart image in the static folder
- Added content for make-transfer and send-deploy commands

### CODE REVIEW

File: source/docs/casper/workflow/deploy-transfer.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+# Transferring Tokens using a Deploy File
+
+This topic explores the use of a deploy file to transfer Casper tokens (CSPR) between accounts on the Casper network. This method of transferring tokens is useful when you want to implement multi-signature deploys. To understand more about multi-signature deploys, see [Two-Party Multi-Signature Deploys](two-party-multi-sig.md). The `make-transfer` command allows you to create a transfer deploy and save the output to a file. You can then have the deploy signed by other parties using the `sign-deploy` command and send it to the network for execution using the `send-deploy` command.
```
Comment (ipopescu) [inline source/docs/casper/workflow/deploy-transfer.md:3]: I have been capitalizing network: "Casper network" to "Casper Network". For consistency, I would capitalize it here too, but we should probably discuss this as a team and create a style guide.

File: source/docs/casper/workflow/deploy-transfer.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+## Prerequisites
+
+You must ensure the following prerequisites are met, before you start using the deploy commands:
+
+1.  Set up your machine as per the [prerequisites](setup.md)
+2.  Set up accounts on the [Testnet](https://testnet.cspr.live/) or [Mainnet](https://cspr.live/)
+1.  Fund the source account
+2.  Get the source *secret key* file path and the target *public key* hex
+3.  Get a valid *node address* from the [Testnet peers](https://testnet.cspr.live/tools/peers) or [Mainnet peers](https://cspr.live/tools/peers) 
+4.  Use the Casper command-line client
```
Comment (ipopescu) [inline source/docs/casper/workflow/deploy-transfer.md:14]: A suggestion to add a link to the command-line client: https://casper.network/docs/workflow/setup#the-casper-command-line-client

File: source/docs/casper/workflow/deploy-transfer.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+The following steps can help you verify if the transfer was successful using the command-line:
+1. [Query the deploy](querying#querying-deploys) to find out the status of execution
+2. [Get the state root hash of the deploy](transfer-workflow#state-root-hash) 
+2. [Query the source account](transfer-workflow#query-the-source-account) to find the main purse `Uref` address of the source account
+3. [Query the target account](transfer-workflow#query-the-target-account) to find the main purse `Uref` address of the target account
+4. [Get the source account balance](transfer-workflow#get-source-account-balance) to check if the amount was deducted from the CSPR balance
+5. [Get the target account balance](transfer-workflow#get-target-account-balance) to confirm that the amount was credited
+
+:::tip
+You can also verify if the transfer was successful by logging on to the https://cspr.live/ website.
```
Comment (ipopescu) [inline source/docs/casper/workflow/deploy-transfer.md:180]: I would add: "... and searching for the deploy hash using the search box."

**Review by ipopescu — APPROVED** [review]
Very nice work, @GloreenS! Thank you!
I've added a few minor suggestions, but the PR is approved.

File: source/docs/casper/workflow/deploy-transfer.md (hunk 3 above)
Comment (GloreenS) [reply source/docs/casper/workflow/deploy-transfer.md:180]: There are multiple ways of viewing your account transactions on cspr.live, so just adding "and checking your account balance", will leave it open for the user on how to view the balance.

**Pull request merged**